bundles/org.openhab.binding.lgthinq/src/main/resources/OH-INF/i18n/lgthinq_xx.properties

```diff
@@ -0,0 +1,30 @@
+# FIXME: please substitute the xx with a proper locale, ie. de
```

Comment:
Please note this comment and rename lgthinq_xx.properties to lgthinq.properties.

Reply:
Sorry. Shame on me :-(